python/pyspark/sql/functions.py

```diff
@@ -189,15 +210,15 @@ def _():
 }
 
 for _name, _doc in _functions.items():
-    globals()[_name] = since(1.3)(_create_function(_name, _doc))
+    globals()[_name] = since(1.3)(ignore_unicode_prefix(_create_function(_name, _doc)))
```

Comment:
And .. the easiest way I think is to take out the examples or not use unicode in the doctests for now as `ignore_unicode_prefix` requires a function.